azure-ts-dynamicresource/index.ts

```diff
@@ -0,0 +1,86 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+import { CDNCustomDomainResource } from "./cdnCustomDomain";
+
+const location = "West US";
```

Comment:
Consider using a configuration parameter for the location instead of a hard-coded value.

Reply:
I'll leave it as-is, but will add a comment that the user should consider using the Pulumi config system to externalize it.

Reply:
from @ericrudder in another example:
>it woudl be great if we could "unhard code these" everywhere ... or at least not grow debt
> it's really a pulumi aniti-pattern, and we should avoid these ... we have a few others (like poor password usage) and they bite us when users copy/paste